.github/workflows/budibase_ci.yml

```diff
@@ -44,7 +44,10 @@ jobs:
           node-version: 14.x
           cache: "yarn"
       - run: yarn
-      - run: yarn nx run-many -t=build --configuration=production
+      # Run build all the projects
+      - run: yarn nx run-many -t=build
+      # Check the types of the projects built via esbuild
+      - run: yarn check:types
```

Comment:
Should we run the type check before the build, in case any artifact is ever produced by the type checking that would interfere with the `dist` dir

Reply:
`yarn check:types` might also produce dist folders, as it will need packages like `string-templates` to be built